README.md

```diff
@@ -451,39 +444,20 @@ See `docs/CHANGELOG.md`.
 
 ## Background
 
-### Why was this fork created?
+### What's the plan and why?
 
-The original Bats repository needed new maintainers, and has not been actively
-maintained since 2013. While there were volunteers for maintainers, attempts to
-organize issues, and outstanding PRs, the lack of write-access to the repo
-hindered progress severely.
+**Tuesday, September 19, 2017:** This was forked from [Bats][bats-orig] at
+commit [0360811][].  It was created via `git clone --bare` and `git push
+--mirror`. See the [Background](#background) section above for more information.
 
-### What's the plan and why?
+[bats-orig]: https://github.com/sstephenson/bats
+[0360811]: https://github.com/sstephenson/bats/commit/03608115df2071fff4eaaff1605768c275e5f81f
+
+This [bats-core repo](https://github.com/bats-core/bats-core) is now the official Bats project.
+
+### Why was this fork created?
 
-The rough plan, originally [outlined
-here](https://github.com/sstephenson/bats/issues/150#issuecomment-323845404) is
-to create a new, mirrored mainline (this repo!). An excerpt:
-
-> **1. Roadmap 1.0:**
-> There are already existing high-quality PRs, and often-requested features and
-> issues, especially here at
-> [#196](https://github.com/sstephenson/bats/issues/196). Leverage these and
-> **consolidate into a single roadmap**.
->
-> **2. Create or choose a fork or *mirror* of this repo to use as the new
-> mainline:**
-> Repoint existing PRs (whichever ones are possible) to the new mainline, get
-> that repo to a stable 1.0. IMO we should create an organization and grant 2-3
-> people admin and write access.
-
-Doing it this way accomplishes a number of things:
-
-1. Removes the dependency on the original maintainer
-1. Enables collaboration and contribution flow again
-1. Allows the possibility of merging back to original, or merging from original
-   if or when the need arises
-1. Prevents lock-out by giving administrative access to more than one person,
-   increases transferability
+The original Bats repository is no longer maintained and write access to it could not be obtained. This fork allowed ongoing maintenance and forward progress for Bats.
```

Comment:
I'm also a little wary of the phrasing "is no longer maintained"... Again, there's a fine distinction between a project officially declared unmaintained by the owner, and one _appearing_ unmaintained from the outside.

Bats upstream is definitely the latter, but only questionably the former. I think we can word this better to be unambiguous without simply declaring the status of upstream by fiat.

Reply:
Good point, I'd agree with this.